routes/per_capita_consumption.py

```diff
@@ -0,0 +1,65 @@
+import logging.config
+from flask import Blueprint, jsonify
+from flask_api import status
+from flask_jwt_extended import get_jwt_identity
+from models.pkv import PKV
+from models.user import User
+from util.database import db
+from util.error import UNKNOWN_USER, NO_PER_CAPITA_CONSUMPTION, exception_message
+from util.login import login_required
+
+
+logger = logging.getLogger(__name__)
+PerCapitaConsumption = Blueprint('PerCapitaConsumption', __name__)
+
+
+def get_moving_average_annualized(meter_id):
+    """ Retrieve the last annualized moving average for the given meter id from the SQLite database.
+    :param str meter_id: the user's meter id
+    :returns: the last PKV together with its timestamp or None if there are no
```

Comment:
```suggestion
    :returns: the last PKV mapped to its timestamp or `None` if there are no
```

Like in [per_capita_consumption](https://github.com/buzzn/mybuzzn-backend/blob/ca4ed21683699e8ecb3c1dac491f182dafc95939/routes/per_capita_consumption.py#L42)